public/schemas/applicant_v5.json.erb

```diff
@@ -15,8 +15,15 @@
         "has_partner_opponent": {
           "type": "boolean"
         },
+        "involvement_type": {
+          "type": ["string", "null"],
+          "enum": ["applicant", null]
+        },
```

Comment:
This feels weird to me.

I'd assume non-required fields would be allowed to be null.

Should we add optional fields to the schema at all?

Reply:
another weird thing... IMO a schema defines what we will accept. How were we accepting submissions with extra fields?!

Reply:
That's what I would expect too.

It looks like the JSON validation isn't doing what we think it's doing (or at least what the code tells me it's doing).

When we [submit values that aren't defined in the schema](https://sentry.io/organizations/ministryofjustice/issues/3702754236/?project=5416052), it passes validation and attempts to create the records —ultimately [raising an unknown attribute error](https://sentry.io/organizations/ministryofjustice/issues/3702754301/?project=5404890).

I'll add a [task to the backlog to investigate](https://dsdmoj.atlassian.net/browse/AP-3583), because I think we should be making use of strict validation when it comes to the JSON schema.

Reply:
It gets even more confusing, because the swagger docs say `involvement_type` should be required 🤔